lib/foreman/dynflow/daemon.rb

```diff
@@ -0,0 +1,71 @@
+require 'fileutils'
```

Comment:
why is it a task? shouldn't it be a full blown daemon / service ?

Additionally, I'm not sure if we should run it using some tool to monitor its health, for example runit.